back/app/controllers/web_api/v1/confirmations_controller.rb

```diff
@@ -15,6 +17,16 @@ def create
 
   private
 
+  def reset_auth_cookie
```

Comment:
Should we also change user token to do this from the backend? Bring everything in line?

Reply:
Yes, I think that would be a good idea. Make the security level a consistent part of the jwt, that's always specified and present.

Then would probably also be good to move the implementation of this to the token class, as opposed to having it in the controller.